src/main/java/com/enonic/xp/app/applications/rest/resource/content/page/part/PartDescriptorIconUrlResolver.java

```diff
@@ -0,0 +1,20 @@
+package com.enonic.xp.app.applications.rest.resource.content.page.part;
+
+import com.enonic.xp.app.applications.rest.resource.schema.IconUrlResolver;
+import com.enonic.xp.icon.Icon;
+import com.enonic.xp.region.PartDescriptor;
+
+
+public final class PartDescriptorIconUrlResolver
+    extends IconUrlResolver
+{
+    public static final String DESCRIPTOR_ICON = "/admin/v2/rest/content/page/part/descriptor/icon/";
```

Comment:
such urls won't work without CS.
Is it just a left-over and we don't use icons?
Maybe instead give null (since it won't be used anyway)